File: tests/openmp_examples/program_control/icv.2.f90
!$omp    masked
!$omp    end masked
!$omp    parallel
!$omp       parallel
!$omp          parallel
!$omp          end parallel
!$omp       end parallel
!$omp    end parallel
!$omp    parallel num_threads(8)
!$omp       parallel
!$omp          parallel
!$omp          end parallel
!$omp       end parallel
!$omp    end parallel
!$omp    parallel num_threads(8,2)
!$omp       parallel
!$omp          parallel
!$omp          end parallel
!$omp       end parallel
!$omp    end parallel

File: tests/openmp_examples/program_control/icv.2.c
#pragma omp begin declare variant match(device={kind(host)})
#pragma omp end declare variant
#pragma omp begin declare variant match(device={kind(nohost)})
#pragma omp end declare variant
#pragma omp masked
#pragma omp parallel
#pragma omp parallel
#pragma omp parallel
#pragma omp parallel num_threads(8)
#pragma omp parallel
#pragma omp parallel
#pragma omp parallel num_threads(8,2)
#pragma omp parallel
#pragma omp parallel
